java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -157,79 +134,269 @@ void executeInstruction(BytecodeCFGBuilder.Instruction instruction) {
       return;
     }
     ProgramState.Pop pop;
+    SymbolicValue sv;
     switch (instruction.opcode) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of non-empty switch cases from 35 to at most 30. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1479)